src/tests/tribol_mfem_common_plane.cpp

```diff
@@ -36,7 +36,7 @@
  * scheme.
  *
  */
-class MfemCommonPlaneTest : public testing::TestWithParam<int> {
+class MfemCommonPlaneTest : public testing::TestWithParam<std::pair<int, tribol::KinematicPenaltyCalculation>> {
```

Comment:
Can we add a zero refinement test on three ranks so we get a zero-element mesh on one of the ranks just as a sanity check?